src/Serilog.Formatting.Elasticsearch/ExceptionAsObjectJsonFormatter.cs

```diff
@@ -42,15 +42,18 @@ public class ExceptionAsObjectJsonFormatter : ElasticsearchJsonFormatter
         /// <param name="formatProvider">Supplies culture-specific formatting information, or null.</param>
         /// <param name="serializer">Inject a serializer to force objects to be serialized over being ToString()</param>
         /// <param name="inlineFields">When set to true values will be written at the root of the json document</param>
+        /// /// <param name="renderMessageTemplate">If true, the message template will be rendered and written to the output as a
```

Comment:
Is the double set of slashes `/// ///` deliberate?

Reply:
No copied from base object, will remove it.